### PR TITLE
Add Privacy Manifest

### PR DESCRIPTION
**Pull request by skyfoxa**
## Description

Siren uses UserDefaults, it has to contain PrivacyInfo manifest, check [Apple doc](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api#4278401)

This manifest will be required [**Starting May 1**](https://developer.apple.com/news/?id=3d8a9yyh)

## Changes
- Update swift-tool to v5.9
- Add UserDefaults privacy manifest

### CODE REVIEW

**Comment by Fl0p** [comment]
+

**Comment by ArtSabintsev** [comment]
I am not really doing iOS development at this time, however, I have a question. If this has to be done at the app level, why does the package need to have the change added?

**Comment by gvizeu** [comment]
It is because Apple requieres by May 1 to add a privacy manifest for all Third-party SDKs
https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_data_use_in_privacy_manifests

**Comment by ArtSabintsev** [comment]
Alright, I'll review it when I'm back in town which will be in early May.

File: Package.swift [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
+            targets: ["Siren"]
+        )
+    ],
+    targets: [
+        .target(
+            name: "Siren",
+            path: "Sources",
+            resources: [
+                .copy("Resources/Siren.bundle"),
+                .process("Resources/PrivacyInfo.xcprivacy")
```
Comment (spigarellisj) [inline Package.swift:20]: Should this just be a copy?

Comment (skyfoxa) [reply Package.swift:20]: there should be no difference, SPM will very likely only `copy` the file. I got inspired by other Package files, for instance here is package file for [Firebase SDK](https://github.com/firebase/firebase-ios-sdk/blob/a2fd7442f454a471f29e2201a08d48120e1a762f/Package.swift#L242)

**Comment by ArtSabintsev** [comment]
Example app doesn't compile.

**Pull request closed without merging**